sql-ballerina/Package.md

```diff
@@ -1,371 +1,14 @@
-## Package overview
+## Package Overview
 
-This Package provides the common interface and functionality to interact with a database. The corresponding database
-clients can be created by using specific database packages such as `MySQL` or using the Java Database Connectivity 
-package `JDBC`.
+The SQL library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
 
-### List of Database Packages
-1. JDBC (Java Database Connectivity) Package
-This package can be used to connect with any database by simply providing the JDBC URL and the other related properties. 
-For more details, see the [JDBC package](https://ballerina.io/learn/api-docs/ballerina/#/jdbc).
+It provides the common interface and functionality to interact with a database. The corresponding database clients can be created by using specific database packages such as MySQL or using the Java Database Connectivity package JDBC.
 
+### Report Issues
 
-2. MySQL Package  
-This package is specially designed to work with a MySQL database and allows to access the functionality 
-provided by MySQL 8.0.x onwards. For more details, see the [MySQL package](https://ballerina.io/learn/api-docs/ballerina/#/mysql).
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

Comment:
```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

Reply:
Change Added in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/2656f12c62276e15a256c133465cb60304d0b041